modules/justfiles/README.md

```diff
@@ -0,0 +1,39 @@
+# `justfiles`
```

Comment:
Maybe some quick link/s on what just & justfiles are can be introduced to the README

Like this link or something else
https://just.systems/man/en/

No need to describe it in detail, just shortly, so image-maintainers know what it's about.

Reply:
Maybe a link to the ublue-related just docs on our site and if there's any just docs on ublue's forum as well.